presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -18,12 +18,32 @@ To configure the Iceberg connector, create a catalog properties file
 ``etc/catalog/iceberg.properties`` with the following contents,
 replacing the properties as appropriate:
 
+Hive Metastore catalog
+^^^^^^^^^^^^^^^^^^^^^^
+
+Iceberg connector supports the same configuration for
+`HMS <https://prestodb.io/docs/current/connector/hive.html#metastore-configuration-properties>`_
+as Hive connector.
+
 .. code-block:: none
 
     connector.name=iceberg
     hive.metastore.uri=hostname:port
     iceberg.catalog.type=hive
 
+Glue catalog
+^^^^^^^^^^^^
+
+Iceberg connector supports the same configuration for
+`Glue <https://prestodb.io/docs/current/connector/hive.html#aws-glue-catalog-configuration-properties>`_
+as Hive connector.
+
+.. code-block:: none
+
+    connector.name=iceberg
+    hive.metastore=glue
+    iceberg.catalog.type=hive
+
```

Comment:
Same as above. Would you move this to the catalog type section? As the Glue catalog is still the Hive metastore catalog, maybe you can just mention it there with the appropriate configurations. Your call.